Day 3 tasks/tasks3.js

```diff
@@ -0,0 +1,174 @@
+console.log(!(4 === '4')); // true
+
+
+// 9. Check for 'on' in "dragon" and "python":
+let py = "python"
+let dr = "dragon"
+console.log(py.includes('on')); // true
+console.log(dr.includes ('on')); // true
+
+
+// 10. Date object activities:
+const today = new Date();
+ console.log(today.getFullYear()); 
+ console.log(today.getMonth() + 1); 
+ console.log(today.getDate()); 
+ console.log(today.getDay()); 
+ console.log(today.getHours()); 
+ console.log(today.getMinutes()); 
+ console.log(today.getTime());
+ 
+ 
+
+//Exercises: Level 2
+
+// 1. Area of a triangle:
+ const base = parseFloat(prompt("Enter base:"));
+ const height = parseFloat(prompt("Enter height:"));
+
+ const area = 0.5 * base * height;
+ alert(`The area of the triangle is ${area}`)
+
+//2. Write a script that prompt the user to enter side a, side b, and side c of the triangle and and calculate the perimeter of triangle (perimeter = a + b + c)
+ const sideA = prompt("Enter side a:");
+ const sideB = prompt("Enter side b:");
+ const sideC = prompt("Enter side c:");
+ const perimeter = Number(sideA) + Number(sideB) + Number(sideC);
+ alert(`The perimeter of the triangle is ${perimeter}`);
+ 
+
+//3. Get length and width using prompt and calculate an area of rectangle (area = length x width and the perimeter of rectangle (perimeter = 2 x (length + width))
+ const length = prompt("Enter length:");
+ const width = prompt("Enter width:");
+ const areaRectangle = length * width;
+ const perimeterRectangle = 2 * (Number(length) + Number(width));
+ alert(`Area: ${areaRectangle}, Perimeter: ${perimeterRectangle}`);
+ 
+
+//4. Get radius using prompt and calculate the area of a circle (area = pi x r x r) and circumference of a circle(c = 2 x pi x r) where pi = 3.14.
+ const radius = prompt("Enter radius:");
+ const pi = Math.PI; // More accurate than hardcoding pi as 3.14
+ const areaCircle = pi * radius * radius;
+
+ const circumferenceCircle = 2 * pi * radius;
+ alert(`Area: ${areaCircle}, Circumference: ${circumferenceCircle}`);
+ 
+
+
+
+// 8. Weekly earnings calculation:
+ const hoursWorked = prompt("Enter hours:");
+ const ratePerHour = prompt("Enter rate per hour:");
+ const weeklyEarnings = hoursWorked * ratePerHour;
+ alert(`Your weekly earning is ${weeklyEarnings}`);
+ 
+// 9 If the length of your name is greater than 7 say, your name is long else say your name is short.
+// 10. Compare your first name length and your family name length and you should get this output.
+
+let firstName = 'Asabeneh'
+let lastName = 'Yetayeh'
+// Your first name, Asabeneh is longer than your family name, Yetayeh
+ 
+
+// 11. Declare two variables myAge and yourAge and assign them initial values and myAge and yourAge.
+  let myAge = 250;
+  let yourAge = 25;
+  alert(`I am ${myAge - yourAge} years older than you.`);
+  
+
+// 12. Using prompt get the year the user was born and if the user is 18 or above allow the user to drive if not tell the user to wait a certain amount of years.
```

Comment:
Are you going to do this task?

Reply:
yes, I' am

Reply:
They were some difficult for me. I'm going to do them later